mungegithub/mungers/publish_scripts/publish_driver.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Oh, I thought this is called. We definitely need a split between prepare_repos and push_repos.